fuse/plugins/micro_physics/yields.py

```diff
@@ -33,6 +33,12 @@ class NestYields(FuseBasePlugin):
 
     save_when = strax.SaveWhen.TARGET
 
+    use_recombination_fluctuation = straxen.URLConfig(
+        default=True,
+        type=bool,
+        help="Turn on or off the recombination fluctuation for beta interactions.",
+    )
```

Comment:
Hi @cfuselli, from looking at the code, `use_recombination_fluctuation` will also work for our pure NEST quanta calculation or? I would change the help message and just call it something like: `"Turn on or off the recombination fluctuation"`